proxy/logging/LogFile.h

```diff
@@ -168,8 +85,6 @@ class LogFile : public LogBufferSink, public RefCountObj
 
   static int write_ascii_logbuffer(LogBufferHeader *buffer_header, int fd, const char *path, const char *alt_format = NULL);
   int write_ascii_logbuffer3(LogBufferHeader *buffer_header, const char *alt_format = NULL);
-  static bool rolled_logfile(char *file);
-  static bool exists(const char *pathname);
```

Comment:
Is it worth taking these out? For client convenience it you could leave them in and have them just forward to the `BaseLogFile` methods. Otherwise you're forcing clients to be aware of the implementation details of `LogFile` (e.g. that it uses `BaseLogFile`).